Shared/Samples/Add web tiled layer/AddWebTiledLayerView.swift

```diff
@@ -0,0 +1,48 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct AddWebTiledLayerView: View {
+    // A map with web tiled layer.
```

Comment:
```suggestion
    /// A map with web tiled layer.
```

For properties, methods and types, we generally use documentation comments, which use 3 slash instead of 2.

Reply:
Thank you for the insight, I appreciate it.